### PR TITLE
Initial Support for egnite querx sensors

### PR DESCRIPTION
**Pull request by Megachip**
Starting on #11261 

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @Megachip 
Could you please rework the YAML Discovery file to be a little bit more "generic": 
- You can filter "QuerxSensorDataType" to identify if a sensor is "Temperature", or something else, using `skip_values` statement.
- You can have a look at `includes/definitions/discovery/vrp.yaml` to see examples of `skip_values`

If you do so, then the type of sensor will be dynamic, instead of your static definition now. 

And you'll have to replace the last number of the num_oid by $index to be completely dynamic : 
- `num_oid: '.1.3.6.1.4.x.x.x.x.x.x.{{ $index }}'`

File: includes/definitions/discovery/querx.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+mib: EGNITEPHYSENSOR-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: querxPhySensorTable
+                    value: querxPhySensorValue
+                    num_oid: '.1.3.6.1.4.1.3444.1.14.1.2.1.5.1'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/querx.yaml:9]: Need to change this to a more dynamic structure, and include `$index`in the num_oid, see comments.

File: tests/snmpsim/querx.snmprec [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|Querx sensor
+1.3.6.1.2.1.1.2.0|6|.1.3.6.1.4.1.3444.1.14
```
Comment (PipoCanaja) [inline tests/snmpsim/querx.snmprec:2]: This test data is not complete (no sensors included). Please regenerate this snmprec file.

**Comment by Jellyfrog** [comment]
Closing due to inactivity. Feel free to reopen

**Pull request closed without merging**